webpack.common.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
       './euth_wagtail/assets/scss/all.scss',
       './euth_wagtail/assets/js/app.js'
     ],
+    captcheck: [
+      './euth/captcha/assets/captcheck.js'
+    ],
```

Comment:
should this have the `dependOn: 'adhocracy4'` as the others?

Reply:
Yes!